src/index.d.ts

```diff
@@ -57,6 +61,23 @@ declare namespace SDKStandardComponents {
         extensionList?: TExtensionList
     }
 
+    type TCredential = {
+        id: string | null;
+        credentialType: 'FIDO';
+        // TODO: Double check if enum is ACTIVE or VERIFIED.
```

Comment:
TODO - are you planning to do in this PR?

Reply:
yes. my mistake. it's supposed to be `ACTIVE`. comment removed.